jenkins-config/managed-config-files/custom/sigul-config/customizedCredentialMappings.sandbox.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 customizedCredentialMappings:
-  - credentialsId: "sigul-nss-pass"
-    tokenKey: "SIGUL_NSS_PASS"
+  - credentialsId: "sigul-config-credentials"
```

Comment:
Looks like the credentials were only created on the prod Jenkins. To do any signing tests, they'll need to be created on the sandbox too.

Reply:
let me get those going.. thanks!